lib/msal-core/src/utils/RequestUtils.ts

```diff
@@ -75,13 +78,40 @@ export class RequestUtils {
         return validatedRequest;
     }
 
+    /**
+     * 
+     * @param request 
+     * @param clientId 
+     * @param interactionType 
+     */
+    static validateLoginRequest(request: AuthenticationParameters, clientId: string, interactionType: InteractionType): AuthenticationParameters {
+        // Check if userRequest is empty
+        if (!request) {
+            request = {};
+        }
+
+        // Check if request scopes are null
+        if (!request.scopes) {
+            request.scopes = [];
+        }
+
+        // Append clientId to scopes by default for login calls
+        request.scopes = ScopeSet.appendScopes(request.scopes, [clientId]);
```

Comment:
clientId again

Reply:
Yeah, this was duplicated code that was removed during the design discussion, I changed it to an array containing openid and profile in the right place.